libs/components/indicators/testing/src/label/label-harness.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { TestBed } from '@angular/core/testing';
+import { SkyLabelType } from '@skyux/indicators';
+
+import { LabelHarnessTestComponent } from './fixtures/label-harness-test.component';
+import { LabelHarnessTestModule } from './fixtures/label-harness-test.module';
+import { SkyLabelHarness } from './label-harness';
+
+async function setupTest(options: { dataSkyId: string }) {
+  await TestBed.configureTestingModule({
+    imports: [LabelHarnessTestModule],
+  }).compileComponents();
+
+  const fixture = TestBed.createComponent(LabelHarnessTestComponent);
+  const loader = TestbedHarnessEnvironment.loader(fixture);
+
+  const labelHarness: SkyLabelHarness = await loader.getHarness(
```

Comment:
I believe you can leave out `: SkyLabelHarness` here as `loader.getHarness()` will get its return type from the provided `SkyLabelHarness.with()` query.